modules/exploits/windows/imap/mercury_login.md

```diff
@@ -0,0 +1,25 @@
+Mercury/32 <= 4.01b contains an stack based buffer overflow in IMAPD LOGIN verb. Sending an specially crafted IMAP login command allows remote code execution.
+
+## Vulnerable Application
+
+        This module exploits a stack buffer overflow in Mercury/32 <= 4.01b IMAPD LOGIN verb. By sending a specially crafted login command, a buffer is corrupted, and code execution is possible. This     	       vulnerability was discovered by (mu-b at digit-labs.org).
+       
+## Info
```

Comment:
Please update to use `Scenarios` header, and show output from a successful exploitation attempt, for consistency with existing modules.

* [Template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)
* [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)